inductiva/molecules/scenarios/protein_solvation/post_processing.py

```diff
@@ -104,13 +84,16 @@ def calculate_rmsf_trajectory(self,
             nglview_visualization: Whether to return visualization of the 
             RMSF using nglview or not."""
         start_time = time.time()
-        universe = self.construct_universe(use_compressed_trajectory)
-        topology_path = os.path.join(self.sim_output_dir, TOPOLOGY_FILE)
+        topology_path = os.path.join(self.sim_output_dir, SYSTEM_TOPOLOGY_FILE)
+        full_trajectory_path = os.path.join(self.sim_output_dir,
+                                            FULL_TRAJECTORY_FILE)
+        full_precision_universe = mda.Universe(topology_path,
+                                               full_trajectory_path)
 
         aligned_trajectory_path = os.path.join(self.sim_output_dir,
                                                "aligned_traj.dcd")
         inductiva.molecules.scenarios.utils.align_trajectory_to_average(
-            universe, aligned_trajectory_path)
+            full_precision_universe, aligned_trajectory_path)
```

Comment:
The PR description mentions that the representation is now done using the compressed trajectory, but this uses the "full precision universe". Can you elaborate on why?

Reply:
Because in this function I am computing the RMSF measure, so I thought that it would be better to use the full precision trajectory to get a more precise measure.

Reply:
Gotcha, thanks.